tests/foreman/ui/test_ldapauthsource.py

```diff
@@ -153,7 +158,6 @@ def test_positive_create_withad_org_and_loc(self):
                         self.ldapauthsource.search(server_name)
                     )
 
-    @stubbed()
     @tier2
     def test_positive_create_withidm_org_and_loc(self):
```

Comment:
nitpick: /s/withidm/with_idm/ . Would be also nice to fix all 'withad' titles as well.